contentScript.js

```diff
@@ -35,5 +46,8 @@ const interval = setInterval(() => {
     console.log("Hiding self...");
     clearInterval(interval);
     hideSelfButton().click();
+    setInterval(() => {
```

Comment:
❌ This will need to be `setTimeout` instead, like the below - otherwise it will keep trying to close it every 2 seconds 😄 

```suggestion
    setTimeout(() => {
```

Reply:
Thanks, this was an obvious error, thanks for catching it!